src/include.am

```diff
@@ -203,6 +203,11 @@ endif
 
 if BUILD_ARMASM
 src_libwolfssl_la_SOURCES += wolfcrypt/src/port/arm/armv8-sha256.c
+if BUILD_ARMASM_INLINE
```

Comment:
this builds the C inline, else builds Asm

Is this correct?

Reply:
Yes. I generate the assembly code as inline C and assembly code.
The files provide the same functions with the same instructions.